src/service/crypto-impl.js

```diff
@@ -215,6 +215,26 @@ export class Crypto {
       data
     ));
   }
+
+  /**
+   * Returns a Uint8Array of the given length full of cryptographically
+   * random values if win.crypto is available.
+   * If win.crypto, is not available, returns null.
+   * @param {number} length The length of the string to return
+   * @return {Uint8Array|null}
+   */
+  getSecureRandomBytes(length) {
+    // Support IE 11
```

Comment:
Surprisingly supported everywhere!

Reply:
Yup! We do need to keep the fallback to window.msCrypto check for IE 11. win.crypto in IE 11 is undefined.